### ISSUE TITLE
New page: How to import data from an existing database

### ISSUE THREAD

**Issue by jamesmosier** [planetscale/docs#30]
Hi 👋 I recently spun up a new database for an existing database I created and I wanted to import that database into my PlanetScale database but it wasn't super clear the best way to do that. I was wondering if there was such a place in the docs and if not if you'd be open to a PR to add a new page to it. I was thinking under the Tutorials section and explaining how you can import a `sql` dump file or connect to a MySQL client and import it through that clients GUI (thanks to John from PS support who helped point me in the right direction).

**Comment by johnyerhot** [comment]
Hi @jamesmosier, this is that John!

For anyone looking both

```
pscale shell <database> <branch> < /path/to/file.sql
``` 

and also 

```
mysql -u <planetscale-db-username> -p<db-password> \
 -h <database-hostname>.psdb.cloud \
 <database-name> < /path/to/sql-dump.sql
```

should work just fine. Just like you can do with any other MySQL database.

But, for the record - yep, 👍 💯  around docs specifying it in our docs too. 

Keep an eye out, we've got something in the works. I'll keep this open until it all ships.

**Comment by tbarn** [comment]
Hey @jamesmosier! 

In case you missed it, we released an imports feature to help out with this in November. 
- More details in the blog post: https://planetscale.com/blog/import-your-mysql-data-to-planetscale
- And docs: https://docs.planetscale.com/concepts/database-imports

If you haven't already been working with the team, I would love to hear if this helps you out.

**Comment by jamesmosier** [comment]
Hi @tbarn! I did see that the new import feature was released but have not had a chance to try it out (I had already manually imported my DB via John's advice).

One thing that was missing previously when the feature was launched was a top-level page in the docs (it was hard to locate these instructions before), but I now see that it is included under the "Getting started with PlanetScale" section. So I think that helps to close this issue. Thank you 💯
